app/views/layouts/_header.html.erb

```diff
@@ -112,19 +112,25 @@
   <div class="modal-dialog modal-lg modal-dialog-scrollable">
     <div class="modal-content">
       <div class="modal-header">
-        <h1 class="modal-title fs-5" id="select_language_dialog_label"><%= t ".select_language" %></h1>
+        <h1 class="modal-title fs-5" id="select_language_dialog_label"><%= t "languages_panes.select_language_list.header.select_language" %></h1>
```

Comment:
What was the logic of moving this string so that it then needed a fully qualified name? I don't it's used in the partial whose scope you've now put it is it?